resources/css/app.css

```diff
@@ -1,18 +1,138 @@
+    --border-color-default: color-mix(in srgb, var(--border, #cbd5e1) calc(100% * 1), transparent);
+    --border-color-default-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --border-color-emphasis: color-mix(in srgb, var(--border-emphasis, #94a3b8) calc(100% * 1), transparent);
+    --border-color-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --background-color-emphasis: color-mix(in srgb, var(--background-emphasis, #e2e8f0) calc(100% * 1), transparent);
+    --background-color: color-mix(in srgb, var(--background, #f1f5f9) calc(100% * 1), transparent);
+    --background-color-muted: color-mix(in srgb, var(--background-muted, #f8fafc) calc(100% * 1), transparent);
+
+    --ring-color-default-emphasis: color-mix(in srgb, var(--border-emphasis, #94a3b8) calc(100% * 1), transparent);
+    --ring-color-default: color-mix(in srgb, var(--border, #cbd5e1) calc(100% * 1), transparent);
+    --ring-color-default-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --ring-color-emphasis: color-mix(in srgb, var(--border-emphasis, #94a3b8) calc(100% * 1), transparent);
+    --ring-color: color-mix(in srgb, var(--border, #cbd5e1) calc(100% * 1), transparent);
+    --ring-color-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --outline-color-default-emphasis: color-mix(in srgb, var(--border-emphasis, #94a3b8) calc(100% * 1), transparent);
+    --outline-color-default: color-mix(in srgb, var(--border, #cbd5e1) calc(100% * 1), transparent);
+    --outline-color-default-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --outline-color-emphasis: color-mix(in srgb, var(--border-emphasis, #94a3b8) calc(100% * 1), transparent);
+    --outline-color: color-mix(in srgb, var(--border, #cbd5e1) calc(100% * 1), transparent);
+    --outline-color-muted: color-mix(in srgb, var(--border-muted, #f1f5f9) calc(100% * 1), transparent);
+
+    --box-shadow-color-default-emphasis: color-mix(in srgb, var(--shadow-emphasis, #64748b) calc(100% * 1), transparent);
+    --box-shadow-color-default: color-mix(in srgb, var(--shadow, #94a3b8) calc(100% * 1), transparent);
+    --box-shadow-color-default-muted: color-mix(in srgb, var(--shadow-muted, #cbd5e1) calc(100% * 1), transparent);
+
+    --box-shadow-color-emphasis: color-mix(in srgb, var(--shadow-emphasis, #0f172a) calc(100% * 1), transparent);
+    --box-shadow-color: color-mix(in srgb, var(--shadow, #334155) calc(100% * 1), transparent);
+    --box-shadow-color-muted: color-mix(in srgb, var(--shadow-muted, #64748b) calc(100% * 1), transparent);
+}
+
+@utility container {
+    margin-inline: auto;
+    padding-inline: 1.25rem;
+}
+
+@utility text {
+    color: var(--text-color);
+}
+@utility text-emphasis {
+    color: var(--text-color-emphasis);
+}
+@utility text-muted {
+    color: var(--text-color-muted);
+}
```

Comment:
With the scoped colors we don't need this, only `text` doesn't work by default so that one should stay. But... it doesn't support the opacity modifier (eg `text/50`), see: https://github.com/tailwindlabs/tailwindcss/issues/16824

For the time being, the "full version" does support it: `text-default/50`